package.nls.json

```diff
@@ -135,5 +135,7 @@
     "cmake-tools.configuration.cmake.additionalKits.description": "Array of paths to custom kit files",
     "cmake-tools.configuration.cmake.exportCompileCommandsFile.description": "Enables exporting compile_commands.json.",
     "cmake-tools.configuration.cmake.useCMakePresets.description": "Use CMakePresets.json to configure drive CMake configure, build, and test. When using CMakePresets.json, kits, variants, and some settings in settings.json will be ignored.",
-    "cmake-tools.configuration.cmake.allowCommentsInPresetsFile.description": "Allow the use of JSON extensions such as comments in CMakePresets.json. Please note that your CMakePresets.json file may be considered invalid by other IDEs or on the command line if you use non-standard JSON."
+    "cmake-tools.configuration.cmake.allowCommentsInPresetsFile.description": "Allow the use of JSON extensions such as comments in CMakePresets.json. Please note that your CMakePresets.json file may be considered invalid by other IDEs or on the command line if you use non-standard JSON.",
+    "cmake-tools.configuration.cmake.launch.allowParallel": "Allow launching multiple instances of the **same target** in parallel.",
```

Comment:
I think we should merge the two settings into a single enum because we only need 3 states, not 4.  Not sure what the new name should be at the moment, but the enum values would be something like "waitForCompletion", "allowParallel", "terminatePrevious".  "waitForCompletion" would be the default value since that is the current behavior today.